controller_manager_msgs/srv/ListControllerInterfaces.srv

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I know it's a PITA but I'd strongly suggest renaming this service. We already have the name "controller interface" for a class, also it doesn't actually capture what's happening here, there's not even a controller at play in this context.

I suggest something along the lines of `ListJointInterfaces` but I wonder if this is strongly required at this stage... I think it may be useful to also denote whether these interfaces are claimed at the moment, perhaps by which controller, etc. I think that from the resource manager PR perspective, this addition is a bit of a feature creep that can lead to a lot more creeping as you can tell by my comment.

Reply:
I suggest: `ListHardwareInterfaces` or `ListResourceInterfaces`

I would leave it am make a new issue with these suggestions. It was very useful for testing of example implementation

Reply:
ec56888

renamed to `ListHardwareInterfaces`